controllers/articles.controllers.js

```diff
@@ -33,3 +34,22 @@ exports.getCommentsByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const data = req.body;
+  const { username } = data;
+
+  Promise.all([
```

Comment:
This logic looks sound but psql will throw an error if we try to create an entry that references a non-existent foreign key